src/sidecar/connectionStatusUtils.test.ts

```diff
@@ -31,6 +32,45 @@ describe("connectionEventHandler", () => {
     sandbox.restore();
   });
 
+  for (const action of [
```

Comment:
New test proving that `reactToCCloudAuthState` gets called whenever we get a CCloud connection state event from sidecar websocket.